project/.github/ISSUE_TEMPLATE.md

```diff
@@ -1,52 +0,0 @@
-<!--
```

Comment:
I think there is a specific way to tell dev-kit to remove a file, something related to name change with DELETE_ prefix. I do not remember exactly. Maybe @OskarStark can help here

Reply:
I guess you're right:
https://github.com/sonata-project/dev-kit/blob/22131a50897dbc52f357a38562d0a475901e09f2/src/Command/DispatchCommand.php#L493-L501

Reply:
Yes we need to prefix, deploy it and add another PR and delete the file from dev-kit